README.md

```diff
@@ -42,6 +42,7 @@ java -jar $ili2pg --schemaimport \
                   --createMetaInfo \                   # creates additional tables with information from the interlis model
                   --createNumChecks \                  # creates constraints for numerical data types
                   --createUnique \                     # creates unique constraints in the db for Interlis unique constraints
+                  --createImportTabs \                 # creates import tables for trace import operations
```

Comment:
I would put this option under "Options that might be relevant dependent on individual settings"